react/client/src/components/Textarea.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/prop-types */
```

Comment:
Yeah, I bet this was annoying 😂  
Is this still giving you errors even though the rule is off?

Reply:
I think it's fine now so I'll get rid of it!